tests/integration/targets/connection_aws_ssm/aws_ssm_integration_test_setup/templates/aws-env-vars.j2

```diff
@@ -1,4 +1,6 @@
 export AWS_ACCESS_KEY_ID={{aws_access_key}}
 export AWS_SECRET_ACCESS_KEY={{aws_secret_key}}
+{% if security_token is defined %}
```

Comment:
I'm getting token errors when testing with the updated CI policies.

```
botocore.exceptions.ClientError: An error occurred (UnrecognizedClientException) when calling the StartSession operation: The security token included in the request is invalid.
fatal: [i-0e7e9edaf5ff18ccb]: FAILED! => {
    "msg": "Unexpected failure during module execution.",
    "stdout": ""
}
```

Reply:
Wrong environment variable :)